src/pilot/assign_role.py

```diff
@@ -1553,7 +1557,8 @@ def main():
             args.role_index,
             args.os_volume_size_gb,
             ironic_client,
-            drac_client)
+            args.node_definition,
```

Comment:
Can you explain the reasoning behind this message signature change?  It looks like passing in drac_client should work.

Reply:
Modified this code 
Please refer https://github.com/dsp-jetpack/JetPack/pull/202